src/screens/Accounts/Accounts.js

```diff
@@ -295,40 +312,114 @@ class AccountsScreen extends React.Component<Props, State> {
         )}
       />
     );
+  }
+
+  renderInitializeAccount(item: NewItem) {
```

Comment:
This seems like almost identical to renderAccount.
We could save some lines by checking one prop to alter the look and action for init card in renderAccount method